src/Database/Migrations/2020-12-28-223112_create_auth_tables.php

```diff
@@ -15,13 +15,18 @@ class CreateAuthTables extends Migration
      */
     private array $tables;
 
+    private array $attributes;
+    private bool $ifNotExists;
+
     public function __construct(?Forge $forge = null)
     {
         parent::__construct($forge);
 
         /** @var Auth $authConfig */
-        $authConfig   = config('Auth');
-        $this->tables = $authConfig->tables;
+        $authConfig        = config('Auth');
+        $this->tables      = $authConfig->tables;
+        $this->ifNotExists = false;
+        $this->attributes  = ($this->db->getPlatform() === 'MySQLi') ? ['ENGINE' => 'InnoDB'] : [];
```

Comment:
@kenjis , as far as I know, **MySQLi**, **OCI8**, **Postgre** and **SQLSRV** accept the attributes `['ENGINE' => 'InnoDB']`.
 And only `SQLite3` does not understand `['ENGINE' => 'InnoDB']`. Isn't it better to act as follows?
```php
$this->attributes = ($this->db->getPlatform() === 'SQLite3') ? [] :  ['ENGINE' => 'InnoDB'];
```

Reply:
No. `InnoDB` (and `ENGINE`) is configration only in MySQL. 
I believe OCI8, Postgre and SQLSRV does not understand the attribute. 
I don't know why these drivers do not report an error like SQLite.

Reply:
So it will stay like i did it, e.g. restricted to MySQL only!